content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -0,0 +1,191 @@
+- The Variables API for [workspace-specific variables](/docs/cloud/api/workspace-variables.html) and [variable sets](/docs/cloud/api/variable-sets.html).
+- The `tfe` provider's [`tfe_variable`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/variable) resource, which can be more convenient for bulk management.
+
+## Workspace-Specific Variables
+
+To view and manage a workspace's variables, go to the workspace and click the **Variables** tab.
+
+The **Variables** page appears, showing all workspace-specific variables and variable sets applied to the workspace. This is where you can add, edit, and delete workspace-specific variables. You can also apply and remove variable sets from the workspace.
+
+![Screenshot: The initial appearance of a workspace's variables page](./images/vars.png)
+
+
+### Add a Variable
+
+To add a variable:
+
+1. Go to the workspace **Variables** page and click **+ Add variable** in the **Workspace Variables** section.
+
+1. Choose a variable type (Terraform or environment), optionally mark the variable as [sensitive](#sensitive-values), and enter a variable key, value, and optional description. Check the **HCL** checkbox to enter a value in HashiCorp Configuration Language.
+
+    Refer to [variable values and format](#variable-values-and-format) for variable limits, allowable values, and formatting.
+
+1. Click **Save variable**. The variable now appears in the list of the workspace's variables and Terraform Cloud will apply it to runs.
+
+  ![Screenshot: A variable being edited](./images/vars-edit.png)
+
+### Edit a Variable
+
+To edit a variable:
+
+1. Click the ellipses next to the variable you want to edit and select **Edit**.
+1. Make any desired changes and click **Save variable**.
+
+
+### Delete a Variable
+
+To delete a variable:
+
+1. Click the ellipses next to the variable you want to delete and select **Delete**.
+1. Click **Yes, delete variable** to confirm your action.
+
+## Variable Sets
+
+> **Hands On:** Try the [Manage Variable Sets in Terraform Cloud tutorial](https://learn.hashicorp.com//tutorials/terraform/manage-variable-sets) on HashiCorp Learn.
+
+To view variable sets for your organization, click **Settings** in the top menu bar, then click **Variable sets** in the left sidebar.
+
+The **Variable sets** page appears, listing all of the organization's variable sets. Click on a variable set to open it and review details about its variables and scoping.
+
+![The variable sets page in the Terraform Cloud UI](link)
+
+### Create Variable Sets
+
+-> **Note:** Variable sets are currently in beta.
+
+To create a variable set:
+
+1. Go to the **Variable Sets** page for your organization and click **Create variable set**. The **Create a new variable set** page appears.
+
+  ![The create a new Variable set page in the Terraform Cloud UI](link)
+
+
+1. Choose a descriptive **Name** for the variable set. You can use any combination of numbers, letters, and characters.
```

Comment:
This list isn't rendering quite right (1 is listed twice), possibly because of the image indenting? If you're in a hurry you can hack it by specifying numbers and sorting out the indenting after launch.

Reply:
Fixed! thank you so much for flagging - I needed to indent the photo a few more spaces...the renderer is so finicky about that.